Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -102,6 +104,22 @@ public GitHub3Plugin() : base(true)
         public override IEnumerable<ISetting> GetSettings()
         {
             yield return OAuthToken;
+
+            var generateTokenLink = new LinkLabel { Text = _generateToken.Text };
+            generateTokenLink.Click += GenerateTokenLink_Click;
+            yield return new PseudoSetting(generateTokenLink);
+        }
+
+        private void GenerateTokenLink_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                Process.Start($"https://{GitHubHost.ValueOrDefault(Instance.Settings)}/settings/tokens/new?description=Token%20for%20GitExtensions&scopes=repo,public_repo");
+            }
+            catch (Exception)
```

Comment:
```
catch
{}
```

Reply:
We must not swallow exceptions, especially those that a user can fix.

Reply:
I don't get your points...
I put a catch here, in case, to avoid a crash if I miss something because I know that `Process.Start()` could throw exception.
But none of these types of exceptions seems to be thrown when opening an url.
They are all related to opening a file or a process.

That's why I did just a basic handling and not something that would add a lot of lines of code for nothing.

I did a small change to display the exception message.
Is it enough or were you thinking about something else?